pie.py

```diff
@@ -15,6 +15,8 @@
 import traceback
 import types
 
+from builtins import str

```

Comment:
? huh, do we need this? I thought builtins were, well, you know, "built in"...

Reply:
The path string being fed into the ctyes function needs to be a Unicode string, wrapping `sys.prefix` in a str() call (that's the new/future string type) was the most conveniant way I could think of. On Py3 this should be a no-op, on Py2 str() should create unicode/future strings and not binary strings.

If there's a better/easier way to get sys.prefix as Unicode across both Py2 & Py3, I'm happy to change, this is the method the cheatsheet recommended.

Reply:
... ah, so, just realised that "future" is a third party package. You must have it in your system site packages, but I just ran the tests on 2.7 and it breaks for me. Will fix with some less-nice but less-dependent-on-other-package means.

Reply:
Turns out it's actually a line LESS:
```
        sys_prefix = sys.prefix if PY3 else unicode(sys.prefix)
```

I'll write a test for this - what was the problem? A space in the path to the pie venv?

Reply:
Damn, didn't realise builtins was part of future.

The original problem was activate was using the short folder name syntax, which stuffs the environment path comparison when trying to decide if the venv is already activated. The shortening now gets undone, but you have to feed a Unicode string to the ctypes Win32 call.

Reply:
OK, getting a test to work was a bit more involved... Basically wrote code to get the SHORT path name from a LONG pathname in the test, injected that into sys.prefix and then verified that PieVenv correctly determined that the venv was already active :-)